jaida/patterns/footer.php

```diff
@@ -34,7 +34,7 @@
 
 <!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"right":"0","left":"0","top":"var:preset|spacing|80","bottom":"var:preset|spacing|70"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
 <div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--80);padding-right:0;padding-bottom:var(--wp--preset--spacing--70);padding-left:0"><!-- wp:paragraph {"style":{"typography":{"fontSize":"0.8rem"}}} -->
```

Comment:
If we move the padding to the inner paragraph that contains the credit, I think we can simplify this.

With block group:
<img width="1269" alt="Screenshot 2024-08-01 at 11 49 56" src="https://github.com/user-attachments/assets/57f732b3-a4e0-4059-bc51-0888669d5510">

Without block group (paragraph only with extra padding):


<img width="1254" alt="Screenshot 2024-08-01 at 11 52 37" src="https://github.com/user-attachments/assets/8c1437f6-1188-4487-a9bb-f290d8b92a2f">